packages/cfd/src/Containers/mt5-trade-modal.tsx

```diff
@@ -46,8 +46,8 @@ const MT5TradeModal = observer(
                 );
             }
             return (
-                <TradeModal
-                    mt5_trade_account={mt5_trade_account}
+                <CTraderDerivXTradeModal
+                    ctrader_derivx_trade_account={mt5_trade_account}
```

Comment:
why is it `mt5_trade_account` here when it's `CTraderDerivXTradeModal`

Reply:
The naming was used at the time of implementation of `mt5_trade_modal` and was not changed since then. I will change it cfds_trader_account